src/spikeinterface/sorters/external/tests/test_singularity_containers.py

```diff
@@ -66,6 +66,12 @@ def test_mountainsort4(run_kwargs):
     print(sorting)
 
 
+def test_mountainsort5(run_kwargs):
+    sorting = ss.run_sorter("mountainsort5", output_folder=cache_folder / "mountainsort5", **run_kwargs)
+    print("resulting sorting")
+    print(sorting)
```

Comment:
@magland can you make a PR with a Dockerfile here: https://github.com/SpikeInterface/spikeinterface-dockerfiles ?

So we can run it through this singularity test :)